bring_api/types.py

```diff
@@ -90,3 +90,34 @@ class BringNotificationsConfigType(TypedDict):
     arguments: List[str]
     listNotificationType: str
     senderPublicUserUuid: str
+
+
+class BringUserSettings(TypedDict):
```

Comment:
if it is a single user setting, I would not use plural, but rather `BringUserSetting` or `BringUserSettingsEntry`